tests/core/test_migrate.py

```diff
@@ -80,7 +81,7 @@ def test_migrate_domain_format_with_required_slots(
     migrated_domain = rasa.shared.utils.io.read_yaml_file(domain_out_file)
 
     migrated_training_data_version = migrated_domain.get("version")
-    assert migrated_training_data_version == '"3.0"'
+    assert migrated_training_data_version == LATEST_TRAINING_DATA_FORMAT_VERSION
```

Comment:
Maybe in order to fix the failing tests in this module, we need to pass in an f-string with the constant? 🤔

Reply:
urg there is always one more 😭 I believe it's another `DoubleQuotedScalarString` thing

Reply:
Yep, in migrate.py there were still two hardcoded references.